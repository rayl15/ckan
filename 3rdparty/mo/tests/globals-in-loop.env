STR=abc
DATA=(111 222)
