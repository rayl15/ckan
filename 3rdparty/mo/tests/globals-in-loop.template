Issue #7
{{STR}}
{{#DATA}}
    Item: {{.}}
    String: {{STR}}
{{/DATA}}
