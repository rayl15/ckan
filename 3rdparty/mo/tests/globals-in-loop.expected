Issue #7
abc
    Item: 111
    String: abc
    Item: 222
    String: abc
